spec/controllers/boards_controller_spec.rb

```diff
@@ -26,39 +26,35 @@
     end
   end
 
-  let(:board) {FactoryGirl.create(:board)}
+  let(:board) {create(:board)}
   describe "GET edit" do
-    it "renders the index template" do
+    it "renders the edit template" do
       get :edit, id: board.id
       expect(response).to render_template(:edit)
     end
   end
 
-  let(:board) {FactoryGirl.create(:board)}
+  let(:board) {create(:board)}
   describe "DELETE destroy" do
     it "delete existing board" do
-      delete :destroy, id: board.id
-
-      expect(Board.count).to eq(0)
+      board
+      expect {delete :destroy, id: board.id}.to change {Board.count}.by(-1)
     end
   end
 
-  let(:board) {FactoryGirl.create(:board)}
+  let(:board) {create(:board)}
   let(:attr) {{id: board.id, title: 'Title updated'}}
   describe "PUT update" do
-    it "update title" do
-      put :update, id: board.id, board: attr
-      board.reload
-      expect(board.title).to eq('Title updated')
+    it "updates title" do
+      expect {put :update, id: board.id, board: attr}.to change {board.reload.title}.to('Title updated')
```

Comment:
nie uzywaj nazywa `attr` bo jest to nazwa uzywana w ruby i przez to Ci zaznacza to na czerwono. Poprostu wybierz inna nazwe ;)